db/migrate/20160415121724_create_cloud_services.rb

```diff
@@ -0,0 +1,21 @@
+class CreateCloudServices < ActiveRecord::Migration[5.0]
+  def change
+    create_table :cloud_services do |t|
+      t.string :ems_ref
+      t.string :source
```

Comment:
I use 'source' for tracking which OpenStack Service this comes from - compute, identity, etc... currently I use only 'compute' but probably it can be omited now. It is needed to pass parameters to Fog so it can connect to correct OpenStack Service (nova-compute, nova-scheduler etc). Currently I have it hardcoded when doing refresh so only 'compute' services will be stored in db but when making calls to Fog to enable/disable scheduling on OpenStack Service I am reading this value from db.